drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/model/synchronizers/impl/ModelSynchronizerImpl.java

```diff
@@ -277,7 +282,7 @@ public void deleteColumn( final BaseColumn column ) throws MoveColumnVetoExcepti
                 final boolean isSalienceUseReverseOrderUpdated = BaseColumnFieldDiffImpl.hasChanged( AttributeCol52.FIELD_REVERSE_ORDER,
                                                                                                      diffs );
                 if ( isSalienceUseRowNumberUpdated || isSalienceUseReverseOrderUpdated ) {
-                    updateSystemControlledColumnValues();
```

Comment:
In fact `updateSystemControlledColumnValues()` calls `systemControlledColumnValuesSynchronizer.updateSystemControlledColumnValues();`. It can suggest that `updateSystemControlledColumnValues()` is no more needed? However it can also suggest nothing. Just my notice.